mustache.mjs

```diff
@@ -647,16 +660,38 @@ Writer.prototype.unescapedValue = function unescapedValue (token, context) {
     return value;
 };
 
-Writer.prototype.escapedValue = function escapedValue (token, context) {
+Writer.prototype.escapedValue = function escapedValue (token, context, config) {
+  var escape = this.getConfigEscape(config) || mustache.escape;
   var value = context.lookup(token[1]);
   if (value != null)
-    return typeof value === 'number' ? String(value) : mustache.escape(value);
+    return (typeof value === 'number' && escape === mustache.escape) ? String(value) : escape(value);
 };
 
 Writer.prototype.rawValue = function rawValue (token) {
   return token[1];
 };
 
+Writer.prototype.getConfigTags = function getConfigTags (config) {
+  if (Array.isArray(config)) {
```

Comment:
Did you consider using our locally defined `isArray()` function like we do elsewhere?

Reply:
Very good point. Fixed :+1: